sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.8 (2024-05-07)
 
+### Features Added
+
+- Added new features for upcoming Event Hubs service release.
```

Comment:
This sentence is fairly generic. Do we want to highlight or specify some of the features, so customers know what's new?

For example, something like: Added support for EventHubs emulators.

Reply:
This sentence is *intentionally* generic.